MAINTAINING.md

```diff
@@ -96,3 +96,401 @@ write effective API documentation, such as setting up the RST files and docstrin
+>  * For the first patch release `1.3.1`, `(x.y.z)-1` is the previous stable `1.3.0`.
+>  * For the minor release `1.3.0`, `(x.y.z)-1` is the last release candidate `1.3.0rc2`.   
+
+Generate the short-form changelog using [this script](https://github.com/Qiskit/qiskit-bot/blob/master/tools/generate_changelog.py) from [the `qiskit-bot` repository](https://github.com/Qiskit/qiskit-bot):
+
+If this is a **_first_ release** scenario, run `generate_changelog.py` with the following parameters:
+
+```bash
+python generate_changelog.py Qiskit/qiskit (x.y.z)-1 -t $MY_GITHUB_API_TOKEN
+```
+
+The default behavior of `generate_changelog.py` is to check for changes on the `main` branch of `upstream`. If you are doing a **_follow-up_ release**, run `generate_changelog.py` using the existing `'stable/x.y'` branch:
+
+```bash
+python generate_changelog.py Qiskit/qiskit (x.y.z)-1 -b 'stable/x.y' -t $MY_GITHUB_API_TOKEN
+```
+    
+In both scenarios, if there are entries under `Missing changelog entry`, label the PRs (the main and the backport) with the `Changelog:<something>` label and repeat `generate_changelog.py` until all the entries have a changelog label and `Missing changelog` section is not shown.
+See [this section](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md#changelog-generation) for more details about the available `Changelog:` labels. 
+
+### 3. Submit a "Prepare x.y.z release" PR
+
+Create a PR that will serve as the commit we tag for the release and label it with `Changelog:None`.
+The PR is like a regular PR in your fork and submitted like the regular PR process.
+This step differs depending on the type of release.
+
+Examples for **_first_ release** PRs:
+
+ * [release-candidate PR for Qiskit 1.3.0rc1](https://github.com/Qiskit/qiskit/pull/13397).
+ * [release-candidate PR for Qiskit 2.0.0rc1](https://github.com/Qiskit/qiskit/pull/13953)
+
+Examples for **_follow-up_ release** PRs:
+
+ * [patch-release PR for Qiskit 2.0.1](https://github.com/Qiskit/qiskit/pull/14339).
+ * [follow-up release-candidate PR for Qiskit 1.3.0rc2](https://github.com/Qiskit/qiskit/pull/13466/).  
+
+
+#### 3.1 Create the `prepare-x.y.z` branch
+
+If this is a **_first_ release**, `stable/x.y` does not exist and you should create the branch out of `main`:
+
+```bash
+git checkout -b prepare-x.y.0rc1 upstream/main
+```
+
+If you are doing a **_follow-up_ release**, make a PR out of the stable branch:
+
+```bash
+git fetch upstream
+git checkout -b prepare-x.y.z upstream/stable/x.y
+```
+
+In both situations, your active branch now is `prepare-x.y.z`.
+
+#### 3.2 Bump version numbers
+
+Once in the `prepare-x.y.z` branch, bump the package version number to `x.y.z` (e.g. `1.4.2` for a patch, or `1.3.0rc2` for a second release candidate).
+The places to update are:
+
+* `qiskit/VERSION.txt`: the only line in the file.
+* `docs/conf.py`: the variables `version` and `release`.
```

Comment:
No need for this anymore as of #14697

Reply:
addressed in 635b5488d6c29da496eb06c72b88293fddde2bd6